### DISCUSSION TITLE
No Startup Menu

### DISCUSSION THREAD

**Discussion by gavinj64738** [frntc/Sidekick64#36] [Q&A]
Heya,

Ive assembled one of these and am getting nothing on the boot screen.  The led 1 and 2 are blinking alternatively.  The Led is showing the boot up screen.  But all i get is the regular c64 start up screen.   The functio and reset switches are rebooting the computer.   The opt button does nothing.   Any suggestions with regards to trouble shooting this?

**Comment by frntc** [comment]
If LEDs are blinking and the TFT/OLED shows a logo then the firmware is booting, and it receives the Phi2 signal. A correct logo hints to a working LVC573.

I would check the LVC07 (it controls the GAME and EXROM lines) and then the two right LVC257 and the LVC245 (i.e. U7 U8 and U2 -- these three, which are placed next to the expansion port connector, are involved in address and data lines).

**Reply by SensoryTrash** [reply]
Hmm i seem to have the same problem wondering if you made any progress on fixing the issue.
